{{cookiecutter.repo_name}}/src/theme/theme.js

```diff
@@ -0,0 +1,22 @@
+import { createMuiTheme } from '@material-ui/core/styles';
+import { red } from '@material-ui/core/colors';
+
+// Create a theme instance.
+const theme = createMuiTheme({
+  palette: {
+    primary: {
+      main: '#556cd6',
+    },
+    secondary: {
+      main: '#19857b',
+    },
+    error: {
+      main: red.A400,
+    },
+    background: {
+      default: '#fff',
+    },
+  },
```

Comment:
We should think about adding some default breakpoints just so there is no question. Kind of thinking like how in foundation that was always listed in _settings file for us to modify or refer to if needed. It's not REALY necessary though